packages/ocaml-manual/ocaml-manual.4.01.0/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+maintainer: "Daniel Bünzli <daniel.buenzl i@erratique.ch>"
+authors: [ "Xavier Leroy"
+           "Damien Doligez"
+           "Alain Frisch"
+           "Jacques Garrigue"
+           "Didier Rémy"
+           "Jérôme Vouillon" ]
+homepage: "http://ocaml.org/"
+doc: "http://caml.inria.fr/pub/docs/manual-ocaml/"
+license: "(c) Institut National de Recherche en Informatique \
+          et en Automatique (INRIA)"
+dev-repo: "https://github.com/ocaml/ocaml.git"
+bug-reports: "http://caml.inria.fr/mantis/"
+available: [ ocaml-version >= "4.01.0" | ocaml-version < "4.02.0" ]
+build: [
+ [ "cp" "-R" "." ocaml-manual:doc "ocaml" {os != "win32"} ]
+ [ "robocopy" "/E" "." ocaml-manual:doc {os = "win32"} ]
```

Comment:
These conditionals need to be for the line rather than the last component in the line (other side of `]`).